build/testdata/024.in

```diff
@@ -2,15 +2,21 @@
 # Single-line tuples are also okay.
 
 str.rewrite(".", "/")
+
 str.rewrite(before=".", after="/")
 
 x = (1,2,3)
+
 x = (1,'2',-3,[],{},(),not b)
 
 y = (1, [2])
+
 y = (1, {2: 3})
+
 y = (1, (2, 3))
+
 y = (1, -(a + b))
+
```

Comment:
why do you modify `.in` files?

Reply:
These files are mostly sequences of one-line tests, I think it makes sense to not test the line feeds formatting in them (there's a special test file for different formatting styles).

Some of these tests will be formatted differently in the two modes in the future (like, 016 and 024 test for tuple formatting), I can split the corresponding `.golden` files into `.build.golden` and `.bzl.golden` instead of touching the input file. But, for example, `043.in` tests for slices which will be formatted the same way, I don't see a reason to complicate it with two different golden files.

Reply:
OK.
If you didn't change the formatting for the build mode, then it's okay.

Reply:
I didn't, it's just the first commit to introduce the difference between the modes for top-level-only files.

Actually, I think it's a good idea to split the `016.golden` and `024.golden` files into two for each mode now, I'll need them to be separate in the near future anyway.

Will revert the `.in` files and create `.build.golden` and `.bzl.golden` instead.